### PR TITLE
Icon for Avocode

### PR DESCRIPTION
**Pull request by Jacajack**
selection_002

Icon for Avocode as requested in https://github.com/numixproject/numix-core/issues/3317

### CODE REVIEW

**Comment by Foggalong** [comment]
I know you're just sticking to the original icon, but we generally avoid gradients like this in icons because it goes against our design guidelines. In this situation I'd probably recommend using a darker green for the background gradient and then a solid light green for the symbol, taking both colours from the gradient in the original icon if possible.

**Comment by Jacajack** [comment]
Sure, give me a second :)

**Comment by Jacajack** [comment]
imageselection_006

I made two version of the icons with swapped colors (I can tweak them a bit If you like). Which one do you prefer?

**Comment by Foggalong** [comment]
I think definitely no gradients and colourful background is the way to go, but maybe we could tweak the symbol a bit so it resembles an avocado a bit more? Something like this  
screenshot_2016-11-06_15-27-09

If you're not up for that just let me know and I'll do the tweaking :+1:

**Comment by Jacajack** [comment]
selection_999 036

Something like this?

**Comment by Foggalong** [comment]
@Jacajack That, with then a slightly smaller version of the outer egg part coloured yellow layered above the light green. Then add a shadow like the main symbol shadow for the stone against the yellow

**Comment by Jacajack** [comment]
selection_999 042

Yeah, that's a real avocado now. At first I wanted the icon to look more or less like the original one, but I like this design even more. :)

**Comment by Foggalong** [comment]
That's looking great! Push that and then I'll do one last code checkover before merging :)

**Comment by Jacajack** [comment]
If you ever need an egg icon, remember about me :stuck_out_tongue_closed_eyes: 
selection_999 043

**Review by Foggalong — CHANGES_REQUESTED** [review]
Excellent, last few nit picky changes now just based on the SVG content rather than the iconography.
- [ ] Increase the size of the light green egg to 20px by 26px with location (14,11). Our standard symbol size is 24px by 24px so this is a nice compromise. Remember to move & resize the background shadow accordingly.
- [ ] Increase yellow eggs size to 18px by 24px with location (15,12). This just keeps everything grid aligned.
- [ ] Same with the stone, increase to 10px by 13px with location (10,15). Remember to keep the shadow for the stone 1px to the right and 1px down, the one pushed is slightly closer than that.
- [ ] When saving the file, save as "Optimised SVG". This means it takes up less file space in packaging but keeps the exact same design

With those done this is good to go :+1:

File: data.json [hunk 1]
```diff
@@ -1032,6 +1032,11 @@
             ]
         }
     },
+	"avocode": {
```
Comment (Foggalong) [inline data.json]: This also has the thing with the tap instead of a space :stuck_out_tongue:

**Review by Foggalong — CHANGES_REQUESTED** [review]

**Comment by Jacajack** [comment]
untitled window_999

Inkscape throws an error when I'm trying to save as optimised SVG. :confused:
Can I use `svgo` or something like this?

<hr>

_Oh... how nice..._
selection_999 044

**Comment by Jacajack** [comment]
After manually retyping some Python, I fixed that. I hope everything is alright :stuck_out_tongue:

**Comment by Foggalong** [comment]
Yeah, it can be a bit messy with that. Thanks for powering through the errors though :)

**Review by Foggalong — APPROVED** [review]

**Pull request merged**